### PR TITLE
[es] feature: getting started - spanish translation

### PR DESCRIPTION
**Pull request by krol3**
Related to https://github.com/open-telemetry/opentelemetry.io/issues/4887

### CODE REVIEW

**Comment by krol3** [comment]
@ramrodo @theletterf @electrocucaracha, could you review please?

File: content/es/docs/getting-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Primeros pasos
+description: Comience con OpenTelemetry según su función.
```
Comment (ramrodo) [inline content/es/docs/getting-started/_index.md:3]:
```suggestion
description: Comienza con OpenTelemetry según su función.
```

File: content/es/docs/getting-started/_index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+no_list: true
+weight: 160
+---
+
+Seleccione un rol[^1] para comenzar:
```
Comment (ramrodo) [inline content/es/docs/getting-started/_index.md:8]:
```suggestion
Selecciona un rol[^1] para comenzar:
```

File: content/es/docs/getting-started/_index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- [Ops](ops/)
+
+</div>
+
+También puedes probar la [demo oficial de OpenTelemetry][demo] para _ver_ cómo es
+la observabilidad con OpenTelemetry.
+
+<div class="l-primary-buttons justify-content-start mt-3 mb-5 ms-3">
+
+- [Pruebe la demostración][demo]
```
Comment (ramrodo) [inline content/es/docs/getting-started/_index.md:22]:
```suggestion
- [Prueba la demostración][demo]
```

File: content/es/docs/getting-started/_index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+</div>
+
+[^1]: Si ninguno de estos roles se aplica a usted, [déjanos saber][].
```
Comment (ramrodo) [inline content/es/docs/getting-started/_index.md:26]:
```suggestion
[^1]: Si ninguno de estos roles se aplica a tí, [déjanos saber][].
```

**Review by ramrodo — COMMENTED** [review]
Some suggestions made

**Review by theletterf — APPROVED** [review]

**Pull request merged**